ci/jenkins/test-vmc.sh

```diff
@@ -0,0 +1,528 @@
+        cat "${CLUSTER_SSHCONFIG}.new" >> "${CLUSTER_SSHCONFIG}"
+    done
+
+    echo "=== Move kubeconfig to control-plane Node ==="
+    control_plane_ip="$(kubectl get nodes -o wide --no-headers=true | awk -v role="$CONTROL_PLANE_NODE_ROLE" '$3 ~ role {print $6}')"
+    ${SSH_WITH_ANTREA_CI_KEY} -n capv@${control_plane_ip} "if [ ! -d ".kube" ]; then mkdir .kube; fi"
+    ${SCP_WITH_ANTREA_CI_KEY} $GIT_CHECKOUT_DIR/jenkins/out/kubeconfig capv@${control_plane_ip}:~/.kube/config
+
+
+    mkdir -p ${GIT_CHECKOUT_DIR}/theia-test-logs
+    go version
+    go mod tidy -compat=1.17
+
+    set +e
+    # HACK: see https://github.com/antrea-io/antrea/issues/2292
+    go test -v -timeout=100m antrea.io/theia/test/e2e --logs-export-dir ${GIT_CHECKOUT_DIR}/theia-test-logs --provider remote --remote.sshconfig "${CLUSTER_SSHCONFIG}" --remote.kubeconfig "${CLUSTER_KUBECONFIG}"
+
+
+    test_rc=$?
+    set -e
+
+    if [[ "$test_rc" != "0" ]]; then
+        echo "=== TEST FAILURE !!! ==="
+        TEST_FAILURE=true
+    else
+        echo "=== TEST SUCCESS !!! ==="
+    fi
+
+    tar -zcf ${GIT_CHECKOUT_DIR}/theia-test-logs.tar.gz ${GIT_CHECKOUT_DIR}/theia-test-logs
+
+}
+
+function cleanup_cluster() {
+    echo "=== Cleaning up VMC cluster ${CLUSTER} ==="
+    export KUBECONFIG=$KUBECONFIG_PATH
+
+    kubectl delete ns ${CLUSTER}
+    rm -rf "${GIT_CHECKOUT_DIR}/jenkins"
+    echo "=== Cleanup cluster ${CLUSTER} succeeded ==="
+
+    release_static_ip
+}
+
+function garbage_collection() {
+    echo "=== Auto cleanup starts ==="
+    export KUBECONFIG=$KUBECONFIG_PATH
+
+    kubectl get ns -l antrea-ci -o custom-columns=Name:.metadata.name,DATE:.metadata.creationTimestamp --no-headers=true | awk '{cmd="echo $(( $(date +%s) - $(date -d "$2" +%s) ))"; cmd | getline t ; print $1, t}' | awk '$1 ~ "matrix" && $2 > 36000 {print $1}' | while read cluster; do
+        # Matrix tests
+        echo "=== Currently ${cluster} has been live for more than 10h ==="
+        kubectl delete ns ${cluster}
+        echo "=== Old namespace ${cluster} has been deleted !!! ==="
+    done
+
+    kubectl get ns -l antrea-ci -o custom-columns=Name:.metadata.name,DATE:.metadata.creationTimestamp --no-headers=true | awk '{cmd="echo $(( $(date +%s) - $(date -d "$2" +%s) ))"; cmd | getline t ; print $1, t}' | awk '$1 ~ "whole-conformance" && $2 > 14400 {print $1}' | while read cluster; do
+        # Whole-conformance test
+        echo "=== Currently ${cluster} has been live for more than 4h ==="
+        kubectl delete ns ${cluster}
+        echo "=== Old namespace ${cluster} has been deleted !!! ==="
+    done
+
+    kubectl get ns -l antrea-ci -o custom-columns=Name:.metadata.name,DATE:.metadata.creationTimestamp --no-headers=true | awk '{cmd="echo $(( $(date +%s) - $(date -d "$2" +%s) ))"; cmd | getline t ; print $1, t}' | awk '$1 !~ "matrix" && $1 !~ "whole-conformance" && $2 > 9000 {print $1}' | while read cluster; do
```

Comment:
Let's delete line 437-450, and change antrea-ci to theia-ci in this line.

We can also change the awk expression in this line from
```
awk '$1 !~ "matrix" && $1 !~ "whole-conformance" && $2 > 9000 {print $1}'
```
to
```
awk '$2 > 9000 {print $1}'
```